### PR TITLE
Added logic to browse to the selected concept and removed some extra …

### PR DESCRIPTION
**Pull request by swaroopsalvi**
Fix to select the concept in concept navigator.

### CODE REVIEW

**Review by dholubek — APPROVED** [review]
looks good

**Comment by dholubek** [comment]
@carldea @swaroopsalvi I think we should hold off on merging to ikmdev/komet/main until the build passes.

File: navigator/src/main/java/dev/ikm/komet/navigator/graph/treetasks/ExpandTask.java [hunk 1]
```diff
@@ -96,39 +94,14 @@ protected Void compute() {
...
-                newTreeTop.setExpanded(true);
-                MultiParentVertexImpl vertexToSelect = pathParentList.get(pathParentList.size() - 1);
-                multiParentGraphViewController.getTreeView().scrollTo(rowNumber);
-                // +1 for the root concept, and +1 for the current?
-                // TODO: figure out why this is consistently off by 2, and update accordingly.
-                int rowToSelect = rowNumber + 2;
-                multiParentGraphViewController.getTreeView().getSelectionModel().select(rowToSelect);
-                multiParentGraphViewController.getTreeView().getSelectionModel().getSelectedItems().getFirst().setExpanded(true);
-            });
+            multiParentGraphViewController.getTreeView().getRoot().getChildren().add(newTreeTop);
```
Comment (carldea) [inline navigator/src/main/java/dev/ikm/komet/navigator/graph/treetasks/ExpandTask.java:98]: Can we add a comment that describes the code below?

**Comment by carldea** [comment]
I added a minor comment. For the most part things look good to me.

File: navigator/src/main/java/dev/ikm/komet/navigator/graph/treetasks/ExpandTask.java (hunk 1 above)
Comment (swaroopsalvi) [reply navigator/src/main/java/dev/ikm/komet/navigator/graph/treetasks/ExpandTask.java:100]: Added the code comments.

**Pull request merged**